pkg/cmd/flags/filter.go

```diff
@@ -456,22 +457,53 @@ func PrepareFilterScopes(filtersArr []string) (*filterscope.FilterScopes, error)
 	return filterScopes, nil
 }
 
-func prepareEventsToTrace(eventFilter cliFilter, setFilter cliFilter, eventsNameToID map[string]events.ID) (map[events.ID]string, error) {
+func prepareEventsToTrace(
+	eventFilter cliFilter, setFilter cliFilter, eventsNameToID map[string]events.ID,
+) (
+	map[events.ID]string, error,
+) {
 	eventsToTrace := eventFilter.Equal
 	excludeEvents := eventFilter.NotEqual
 	setsToTrace := setFilter.Equal
 
 	var res map[events.ID]string
 	setsToEvents := make(map[string][]events.ID)
 	isExcluded := make(map[events.ID]bool)
+
+	// build a map: k:set, v:eventID
 	for id, event := range events.Definitions.Events() {
 		for _, set := range event.Sets {
 			setsToEvents[set] = append(setsToEvents[set], id)
 		}
 	}
+
+	// Exclude network events from the default set if kernel v4.19.
+	// Issue: https://github.com/aquasecurity/tracee/pull/2771
+	// TODO: workaround until we don't have fetures probe feature.
+	if osInfo, err := helpers.GetOSInfo(); err == nil {
+		kernel51ComparedToRunningKernel, err := osInfo.CompareOSBaseKernelRelease("5.1.0")
+		if err != nil {
+			logger.Error("failed to compare kernel version", "error", err)
+		}
+		if kernel51ComparedToRunningKernel == helpers.KernelVersionNewer {
```

Comment:
It is being compared even when `osInfo.CompareOSBaseKernelRelease()` may have returned an error.

Reply:
true. fixed.